docker/server/src/main/common/image-bootstrap.properties

```diff
@@ -0,0 +1,11 @@
+Configuration.rootFile=grpc-api.prop
+io.deephaven.configuration.PropertyInputStreamLoader.override=io.deephaven.configuration.PropertyInputStreamLoaderTraditional
+LoggerFactory.silenceOnProcessEnvironment=true
+workspace=.
+devroot=.
+stdout.toLogBuffer=true
+stderr.toLogBuffer=true
+
+jpy.jpyLib=/usr/local/lib/python3.7/dist-packages/jpy.cpython-37m-x86_64-linux-gnu.so
```

Comment:
this file could be modified when copied, such that the arch and py version can be parameterized by a future map.

definitely not requesting that you try arm, just requesting that the arg be made available, plumbed through the gradle/docker barrier so we have one place to change, as long as we're already providing the ability to parameterize the server build.

Reply:
I'd actually like to parameterize for architecture first before trying to architecture for python version; and before parameterizing for python version, I think we should work on building up the stack as a virtual environment.

This bootstrap file is meant to aid in just the docker image building process; I expect we'll need to parameterize our native run instructions more explicitly to pick up python.

Reply:
Parameterizing for arch outside of properties like this is only an extra 1-3 lines in Docker.groovy - with the other images moved out externally (and already building aarch64), this should be the last piece we need to fix.